<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newProfileDialog</class>
 <widget class="QDialog" name="newProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>511</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>RoboBot - New Profile</string>
  </property>
  <widget class="QLabel" name="clProfileNameLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Profile Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="clProfileNameLineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Profile 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="clPostDetailLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Craiglist Post Detail</string>
   </property>
  </widget>
  <widget class="QComboBox" name="clSubLocationComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>270</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="clLocationLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Location:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="clLocationComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>270</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="clSubLocationLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sub Location:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clCreateProfileSaveBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QComboBox" name="clPostAttrComboBox_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>270</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="clPostAttrLbl_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Post Attribute #2:</string>
   </property>
  </widget>
  <widget class="QLabel" name="clPostAttrLbl_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Post Attribute #4:</string>
   </property>
  </widget>
  <widget class="QLabel" name="clPostAttrLbl_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Post Attribute #3:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="clPostAttrComboBox_1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>270</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="clPostAttrComboBox_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>270</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="clPostTypeLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Post Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="clPostAttrLbl_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Post Attribute #5:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="clPostAttrComboBox_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>270</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="clPostTypeComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>270</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="clPostAttrLbl_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Post Attribute #1:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="clPostAttrComboBox_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>270</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
